### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by cpaasch**
PR for @matttbe

### CODE REVIEW

File: draft-schedulers.mkd [hunk 1]
```diff
@@ -173,10 +173,12 @@ which render the re-assembly superfluous or lowering at least its effort.
 In this document, we document a series of multipath packet schedulers that
```
Comment (matttbe) [inline draft-schedulers.mkd:173]: did we really document that? :thinking:

File: draft-schedulers.mkd [hunk 2]
```diff
@@ -173,10 +173,12 @@ which render the re-assembly superfluous or lowering at least its effort.
...
 are known to provide performance that matches well the requirements of specific
 applications. To describe these packet schedulers, we assume an
-abstract transport that is briefly presented in {{abstract}}. We then describe
-the different schedulers in {{schedulers}}. To keep the description as simple
-and intuitive as possible, we assume here multipath connections that are
-composed of two paths, a frequent deployment scenario for multipath transport.
+abstract transport that is briefly presented in {{abstract}}. In {{challenges}}
+we describe the challenges and constraints around a multipath scheduler.
```
Comment (matttbe) [inline draft-schedulers.mkd:177]: good catch for the missing parts!

File: draft-schedulers.mkd [hunk 3]
```diff
@@ -411,8 +409,8 @@ receive-window, this scheduler deploys a fast reinjection mechanism to quickly
 recover from the head-of-line blocking.
 
 At each round, the scheduler iterates over the list of paths that are eligible
-for transmission. To decide whether or not a path a few conditions need to be
-satisfied:
+for transmission. To decide whether or not a path is active, a few conditions
```
Comment (matttbe) [inline draft-schedulers.mkd]: I am not sure about the word "active" which sounds here like "is currently used". I guess we want to say more something like: "can be used". Or "is eligible". No?

**Review by matttbe — CHANGES_REQUESTED** [review]
thank you for these changes. I just have one minor question about "active" ;-)

File: draft-schedulers.mkd (hunk 3 above)
Comment (cpaasch) [reply draft-schedulers.mkd]: yep, is eligible is better

**Review by matttbe — APPROVED** [review]
thank you for the quick update!

**Pull request merged**